README.md

```diff
@@ -66,56 +66,23 @@ A bit more about the different fields...
 ## Available Actions
 
 ### Default
-The `jbi.actions.default` action will create or update the Jira issue and its comments.
-It will also set the Jira issue URL in the Bugzilla bug `see_also` field.
+The `jbi.actions.default` action will take the list of steps to be executed when
+the Webhook is received from configuration.
+When none is specified, it will create or update the Jira issue, publish comments when
+assignee, status, or resolution are changed, or when a comment is posted on the Bugzilla ticket.
```

Comment:
> publish comments when assignee, status, or resolution are changed

Does this actually publish a comment? Or would this fall under "create or update the Jira issue"?

Reply:
With the default action, JBI posts a comment on the Jira when assignee, status, or resolution is changed on the Bugzilla ticket.

With the default_with_assignee_and... one, when these fields change on Bugzilla, JBI set them on the Jira issue, but does not post any comment.

Reply:
Ahh interesting. Thanks for the clarification.